src/sentry/static/sentry/app/actionCreators/plugins.jsx

```diff
@@ -0,0 +1,47 @@
+import PluginActions from '../actions/pluginActions';
+import IndicatorStore from '../stores/indicatorStore';
+
+function doUpdate(api, {orgId, projectId, pluginId, update: updateObj, ...params}) {
```

Comment:
`...params` is just the method. It took me a while to realise what it was since we pass `...params` around a lot. I'd suggest just passing a boolean for `update`? rather than `updateObj` and `...params`

Reply:
I was definitely overthinking this a bit, but the idea for `updateObj` is so that it can accept updates for any plugin data (e.g. configuration). `doUpdate` is consuming the arguments it needs from `params` and forwarding the rest to the API client, but since it's only `data` and `method` let's be explicit with the params.